src/pages/Signup.tsx

```
@@ -0,0 +1,230 @@
+      const finalForm: SignupRequestType = {
+        mail,
+        password,
+        interest,
+      };
+      console.log("회원가입 요청", finalForm);
+
+      try {
+        await signup(finalForm);
+        navigate("/jobs");
+        isLogin();
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)